tests/command_count_test.go

```diff
@@ -0,0 +1,41 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCOUNT(t *testing.T) {
+	commandsCount := 40
+
+	subscriber := getLocalConnection()
+
+	responseValue := fireCommand(subscriber, "COUNT")
+	responseCount  := responseValue.(int64)
+	if responseValue == nil {
+		t.Fail()
+	}
+
+	assert.Assert(t, responseCount == int64(commandsCount),
+		fmt.Sprintf("Unexpected number of CLI commands found. %d expected, %d found", commandsCount, responseCount))
+}
+
+func call(howmany int, t *testing.B) {
+	subscriber := getLocalConnection()
+	for i := 0; i < howmany; i++ {
+		rp := fireCommandAndGetRESPParser(subscriber, "COUNT")
+		if rp == nil {
+			t.Fail()
+		}
+	}
+}
+
+func BenchmarkCountCommand200(t *testing.B) {
+	call(200, t)
+}
+
+func BenchmarkCountCommand2000(t *testing.B) {
+	call(2000, t)
+}
```

Comment:
What is the purpose of passing `200` or `2000` here? Should it just keep firing the `COMMAND COUNT` command, essentially not needing the inner for loop?

Also, the name of the command is `COMMAND COUNT` and not `COUNT`. Please go through the Redis documentation linkedin the issue description.

Reply:
Updated the command name to COMMAND COUNT;